WooCommerce/Classes/ViewRelated/Coupons/UsageDetails/CouponUsageDetailsViewModel.swift

```diff
@@ -0,0 +1,72 @@
+import Combine
+import Foundation
+import Yosemite
+
+/// View Model for `CouponUsageDetails`
+///
+final class CouponUsageDetailsViewModel: ObservableObject {
+    private let coupon: Coupon
+    private let currencySettings: CurrencySettings
+
+    var currencySymbol: String {
+        currencySettings.symbol(from: currencySettings.currencyCode)
+    }
+
+    @Published var minimumSpend: String = ""
+
+    @Published var maximumSpend: String = ""
+
+    @Published var usageLimitPerCoupon: String = ""
+
+    @Published var usageLimitPerUser: String = ""
+
+    @Published var limitUsageToXItems: String = ""
+
+    @Published var allowedEmails: String = ""
+
+    @Published var individualUseOnly: Bool = false
+
+    @Published var excludeSaleItems: Bool = false
```

Comment:
nit: I wonder if it makes sense to initialize these values in the initializer so that it's easier to unify the default values?

Reply:
Makes sense! Updated in 8e7d0c36d30392d9e0b928188fdbc99cc99defe0.